gui/default/vendor/bootstrap/css/bootstrap.css

```diff
@@ -1649,7 +1649,7 @@ input[type="search"] {
 input[type="radio"],
 input[type="checkbox"] {
   margin: 4px 0 0;
-  margin-top: 1px \9;
```

Comment:
Why touch this one? The whole `\9` thing is an old hack for IE 7/8/9 (see https://stackoverflow.com/questions/8004765/what-is-the-meaning-of-9-in-this-width-property-value-declaration). It should be irrelevant for other browsers.